misc/services/mobile-webui/mobile-webui-frontend-testing/tests/spec/picking/picking_catchWeight.spec.js

```diff
@@ -42,21 +42,38 @@ const createMasterdata = async ({ showLastPickedBestBeforeDateForLines } = {}) =
                     uomConversions: [{ from: 'PCE', to: 'KGM', multiplyRate: 0.10, isCatchUOMForProduct: true }],
                     prices: [{ price: 5, uom: 'KGM', invoicableQtyBasedOn: 'CatchWeight' }]
                 },
+                "P2": {
```

Comment:
pls avoid overcomplicating the masterdata. that would lead to unmaintainable/not understandable/hard to read tests.